src/schedule/ll_schedule.c

```diff
@@ -114,7 +114,7 @@ static void schedule_ll_tasks_execute(struct ll_schedule_data *sch,
 			atomic_sub(&sch->num_tasks, 1);
 			if (!atomic_read(&sch->num_tasks))
 				sch->domain->registered[cpu] = false;
-			tr_info(&ll_tr, "task complete %p %s", (uintptr_t)task,
+			tr_info(&ll_tr, "task complete %p %x", (void *)task,
```

Comment:
Does the current trace system support passing (void *) parameters without build failures?